components/shared/compositing/Cargo.toml

```diff
@@ -11,19 +11,19 @@ name = "compositing_traits"
 path = "lib.rs"
 
 [dependencies]
-canvas = { path = "../canvas" }
+canvas = { path = "../../canvas" }
 crossbeam-channel = { workspace = true }
-embedder_traits = { path = "../embedder_traits" }
+embedder_traits = { workspace = true }
 euclid = { workspace = true }
-gfx_traits = { path = "../gfx_traits" }
+gfx_traits = { workspace = true }
 ipc-channel = { workspace = true }
 keyboard-types = { workspace = true }
 log = { workspace = true }
-msg = { path = "../msg" }
-net_traits = { path = "../net_traits" }
-profile_traits = { path = "../profile_traits" }
-script_traits = { path = "../script_traits" }
-servo_url = { path = "../url" }
-style_traits = { path = "../style_traits" }
+msg = { workspace = true }
+net_traits = { workspace = true }
+profile_traits = { workspace = true }
+script_traits = { workspace = true }
+servo_url = { path = "../../url" }
+style_traits = { workspace = true }
```

Comment:
Same comment as above i.e accidentally importing types with `feature = servo` enabled in workspace.